task_3_1/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+)
+
+type Person struct {
+	firstName string
+	lastName  string
+	birthday  time.Time
+}
+
+type People []Person
+
+var dateLayout string = "2006-01-02"
+
+func (p People) Len() int {
+	return len(p)
+}
+
+func (p Person) String() string {
+	return fmt.Sprintf("%v, %v, %v \n", p.firstName, p.lastName, p.birthday.Format(dateLayout))
```

Comment:
Generally it's not advised to add "\n" to stringer func. E.g. one might want to add name inside of a phrase, like "Hello, %s, welcome to the web", and formatting will be broken.

Reply:
other mentor (Ilya) also mentioned this. But what to use instead?